app/players/dwarves/dwarf_blacksmith.py

```diff
@@ -0,0 +1,21 @@
+from .dwarf import Dwarf
+
+
+class DwarfBlacksmith(Dwarf):
+    def __init__(
+            self,
+            nickname: str,
+            favourite_dish: str,
+            skill_level: int
+    ) -> None:
+        super().__init__(nickname, favourite_dish)
+        self.skill_level = skill_level

```

Comment:
Make sure that you store arguments in the protected attributes as mentioned in the checklist

Reply:
not resolved